frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/TagEditorParam.styled.tsx

```diff
@@ -17,12 +17,13 @@ export const TagName = styled.h3`
 
 interface ContainerLabelProps {
   paddingTop?: boolean;
+  smallPaddingBottom?: boolean;
 }
 
 export const ContainerLabel = styled.div<ContainerLabelProps>`
   display: block;
   font-weight: 700;
-  padding-bottom: 0.5rem;
+  padding-bottom: ${props => (props.smallPaddingBottom ? "0.25rem" : "0.5rem")};
```

Comment:
We don't need `smallPaddingBottom` at all if we use `margin-bottom` instead of `padding-bottom`:

```suggestion
  margin-bottom: 0.5rem;
```

Reply:
Yeah, right! I'll address in the next PR to avoid waiting for the tests.

Reply:
Addressed here: https://github.com/metabase/metabase/pull/38997/commits/79f11286dd88a3ad04378bc9a5e73365aa12f3cb